### PR TITLE
[ADD] Company Registry Module

### PR DESCRIPTION
**Pull request by andreschenkels**
Add a company registry option like on res_company

### CODE REVIEW

File: partner_company_registry/README.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+========================
+Partner Comapny Registry
```
Comment (hbrunn) [inline partner_company_registry/README.rst:6]: typo

File: partner_company_registry/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    'author': "ICTSTUDIO, "
+              "Odoo Community Association (OCA)",
+    'maintainer': 'ICTSTUDIO',
+    'license': 'AGPL-3',
+    'website': 'http://www.ictstudio.eu',
+    'depends': [
+        'base',
+    ],
+    'data': [
+        'view/res_partner.xml',
```
Comment (hbrunn) [inline partner_company_registry/__openerp__.py:17]: the folder is called `viewS`

File: partner_company_registry/views/res_partner.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="0">
```
Comment (hbrunn) [inline partner_company_registry/views/res_partner.xml:3]: remote attribute with default value

File: partner_company_registry/views/res_partner.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+        <record model="ir.ui.view" id="view_partner_form_partner_company_registry">
+            <field name="name">partner.view.form.partner_company_registry</field>
```
Comment (hbrunn) [inline partner_company_registry/views/res_partner.xml:6]: remove this field, odoo generates a name for you

**Review by hbrunn — CHANGES_REQUESTED** [review]

File: partner_company_registry/README.rst [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+------------
+
+* André Schenkels <a.schenkels@ictstudio.eu>
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
```
Comment (petrus-v) [inline partner_company_registry/README.rst:25]: If I remember new guide lines required https links

**Review by petrus-v — CHANGES_REQUESTED** [review]

**Comment by rafaelbn** [comment]
Hi @aschenkels-ictstudio , are you going to continue this PR?  commented on 2 Dec 2016 Let us know please

**Comment by rafaelbn** [comment]
7 months without answer

**Pull request closed without merging**